README.md

```diff
@@ -15,18 +15,19 @@ juju deploy sdcore-nms-k8s --channel=edge
 juju deploy traefik-k8s --trust --config external_hostname=<your hostname> --config routing_mode=subdomain
 juju deploy sdcore-upf-k8s --channel=1.5/edge --trust
 juju deploy mongodb-k8s --trust --channel=6/beta
-juju deploy sdcore-webui-k8s --channel=1.5/edge
 juju deploy sdcore-gnbsim-k8s --trust --channel=1.5/edge
-juju integrate mongodb-k8s sdcore-webui-k8s
+juju deploy grafana-agent-k8s --trust --channel=latest/stable
```

Comment:
For line 9, don't we need to use `juju deploy sdcore-nms-k8s --channel=1.5/edge` ?